src/main/features/dashboard/views/macro/claimStatus/noReponse.njk

```diff
@@ -0,0 +1,35 @@
+{% from "timeRemaining.njk" import timeRemaining %}
+
+{% macro noResponseForClaimantDashboard() %}
+  {{ t('Your claim has been sent.') }}
+{% endmacro %}
+
+{% macro noResponseForClaimantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('Your claim has been sent') }}</h2>
+  <p>
+    {{ t('{{ defendantName }} has until {{ responseDeadline }} to respond. They can request an extra 14 days if they need it.',
+      { defendantName: claim.claimData.defendant.name, responseDeadline: claim.responseDeadline | date }) }}
+  </p>
+{% endmacro %}
+
+{% macro noResponseForDefendantDashboard(claim) %}
+  {{ t('Respond to claim.') }}
+  <span class="block mobile-inline font-xsmall">
+    {{ timeRemaining(claim.remainingDays, isAfter4pm()) }}
+  </span>
+{% endmacro %}
+
+{% macro noResponseForDefendantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('You haven’t responded to this claim') }}</h2>
+  <p>
+    {{ t('You need to respond before 4pm on {{ responseDeadline }} {{ remainingTime }}.',
+      { responseDeadline: claim.responseDeadline | date, remainingTime: timeRemaining(claim.remainingDays, isAfter4pm()) }) | safe }}
+  </p>
+  <p>
+    {{ _respondToClaimLink(claim) }}
+  </p>
+{% endmacro %}
+
+{% macro _respondToClaimLink(claim) %}
+  <a href="{{ ResponsePaths.taskListPage.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Respond to claim') }}</a>
```

Comment:
Not clickable in dashboard
REQ

Reply:
Inside the table you mean? We should not have no links in the table except `Continue with claim`